usim/py/exceptions.py

```diff
@@ -0,0 +1,40 @@
+"""
+This module defines the exceptions exclusive to the SimPy compatibility layer.
+These are used as shutdown signals between the environment and processes.
+Note that with the exception of :py:exc:`~usim.py.exceptions.Interrupt`,
+all signals should be considered internal and not meant to be handled manually.
+"""
+
+
+class NotEmulatedError(NotImplementedError):
+    """An operation of the 'simpy' API is not emulated by the 'usim.py' API"""
+
+
+class StopSimulation(BaseException):
+    """Signal to stop a simulation"""
+
+
+class SimPyException(Exception):
+    """Base case for all exceptions of ``usim.py`` that can safely be handled"""
```

Comment:
What happens to exceptions from SimPy that cannot be handled safely?

Reply:
The ``StopSimulation`` (and ``NotEmulatedError ``) is not supposed to be handled manually. How bout this?
```suggestion
    """Base case for all exceptions of ``usim.py`` that can safely be handled in a simulation"""
```

Reply:
You could also say *during a simulation*. Change as you prefer :)